health-services/project/src/main/resources/db/migration/main/V20221208135800__project_staff_ddl.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE PROJECT_STAFF
+(
+    id                character varying(64),
```

Comment:
id NOT NULL  PRIMARY KEY
tenantId NOT NULL
projectId NOT NULL
staffId NOT NULL